bootstrap/kernel_bootstrap.php

```diff
@@ -25,9 +25,18 @@
 }
 
 $xml = new \SimpleXMLElement(file_get_contents($phpUnitFile));
-$kernelDir = $xml->php[0]->server[0]['value'];
 
-$kernelFile = $rootDir.'/'.$kernelDir.'/AppKernel.php';
+$envDir = $xml->xpath("//php/server[@name='KERNEL_DIR']");
```

Comment:
i thought phpunit will provide this as environment variable, no? this code is more correct than the previous, but i am not sure we needed to read the xml file previously

Reply:
phpunit will catch these values, yes. but also the same values are used in `bootstrap/kernel_bootstrap.php` (this file is included from `bin\console`) and without environment variables app kernel could not be instantiated

this PR reads `phpunit.xml.dist` for console commands and `bootstrap/kernel_bootstrap.php` instantiate `KERNEL_CLASS` from `KERNEL_DIR`